ckanext/ontario_theme/templates/internal/datastore/dictionary.html

```diff
@@ -8,7 +8,7 @@
 
 {% block pre_primary %}
   {% set back_url = h.url_for(pkg.type ~ '_resource.edit', id=pkg.name, resource_id=res.id) %}
```

Comment:
Can we change this `back_url` to the correct step 2 (successful validation page)?

Reply:
We should not be directing back to Step 2, which is an intermediate page, the user should be directed back to Step 1.